.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+updates:       
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    commit-message:
+      prefix: "[GitHub Actions] "
```

Comment:
This prefix is not visible in the fork PRs, has it been added after the PRs were created or it doesn't work perhaps?

Reply:
Guess I added it afterwards. I'll let the PR be recreated

Reply:
Here is one created later: https://github.com/sgiehl/matomo/pull/14